blocks/procedures.js

```diff
@@ -468,13 +453,28 @@ Blockly.Blocks['procedures_mutatorcontainer'] = {
   }
 };
 
+var parameterCounter = 1;
+
 Blockly.Blocks['procedures_mutatorarg'] = {
   /**
    * Mutator block for procedure argument.
    * @this Blockly.Block
    */
   init: function() {
-    var field = new Blockly.FieldTextInput('x', this.validator_);
+    var paramName = "param" + parameterCounter.toString();
```

Comment:
Instead of changing the default name of the block it should be changed in the copies of the block when they're added to the mutator workspace. This will keep the behavior consistent with how new functions get named as they're dragged onto the workspace.